src/editor-previewer/editor-previewer.component.scss

```diff
@@ -1,7 +1,7 @@
 div.preview-container {
   width: 100%;
+  height: 95vh;
```

Comment:
I think we should use `height: 100%;` instead. If you think it's better to do later, we should create an issue then.